mcodingbot/tasks/stats.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import annotations
+
+import asyncio
+import traceback
+from dataclasses import dataclass
+from math import log
+from typing import TYPE_CHECKING, Any
+
+from mcodingbot.config import CONFIG
+
+if TYPE_CHECKING:
+    from mcodingbot.bot import Bot
+
+
+async def loop_update_channels(bot: Bot) -> None:
+    while True:
+        try:
+            await update_channels(bot)
+        except Exception:
+            print("Failed to update channel stats:")
```

Comment:
use the logging module. I dont think this should be here anyway tbh. Errors should be handled in a way thats more abstracted and more detailed. (use a `YouTubAPI` class because that will deal with the problem).

Reply:
I'm not willing to re-figure out how to use youtube's garbage api or their thingy. This code is just copy+pasted from mCodingBot-pincer with a few modifications

Reply:
Im not happy with this code quality. I'm gonna make a PR to fix this later then.

Reply:
I made it use LOGGER, so that's better.